src/javascripts/components/blog/ArticlePreview.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { Link } from 'react-router'
+
+class ArticlePreview extends React.Component {
+  render() {
+    return (
+      <Link to="/article"><div className="articlePreview">
+        <div className="--imageContainer">
```

Comment:
I don't think you can use `--` at the beginning of a class.  CSS has `--` reserved for something else.  You might have to make that `-imageContainer`

Also -- our standard is `kabob-case` for class names, not `camelCase`